tests/framework/helpers/UrlHelperTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace yiiunit\framework\helpers;
+
+use yii\helpers\Url;
+use yiiunit\TestCase;
+
+/**
+ * StringHelperTest
+ *
+ * @group helpers
+ */
+class UrlHelperTest extends TestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->mockApplication();
```

Comment:
You only test one static method, maybe you don't need the whole application mock for it?